### DISCUSSION TITLE
[Community Plugin] RapidAPI yh-finance seems delisted from the platform

### DISCUSSION THREAD

**Discussion by billpwchan** [lowlighter/metrics#1123] [Q&A]
For the stock plugin, it uses the API provided in api.com/apidojo/api/yh-finance but seems this API is no longer available to the public.

**Comment by lowlighter** [comment]
I think maybe they delisted it because they seems to have their own api now on https://www.yahoofinanceapi.com/

I think `https://yh-finance.p.rapidapi.com/stock/v2/get-chart` may be replaced by `/v8/finance/chart/{ticker}` (seems to have same or similar output), while `https://yh-finance.p.rapidapi.com/stock/v2/get-profile` could be replaced with `/v11/finance/quoteSummary/{symbol}` without too much trouble

But in any case the API token would need to be changed since it's not the same service

I haven't checked, but is the API down or just unlisted ? Not sure if we should support both endpoints
